javascript/builtins/Array.json

```diff
@@ -535,9 +535,18 @@
               },
               "edge": "mirror",
               "firefox": {
+                "version_added": "103",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": "javascript.options.experimental.array_find_last",
+                    "value_to_set": "true"
+                  }
+                ]
+              },
+              "firefox_android": {
                 "version_added": false
               },
-              "firefox_android": "mirror",
```

Comment:
@queengooborg FYI, my assumption is that "mirror" is not smart enough to know that android shouldn't mirror preferenced versions. So removing and marking as false.

Reply:
It should do the right thing actually:

https://github.com/mdn/browser-compat-data/blob/b53950916c1aeba50dd0fdcc0f6a9395aed6615b/scripts/release/mirror.ts#L354-L357

Reply:
That's actually pretty cool! I'll revert those.